DESCRIPTION

```diff
@@ -34,7 +34,7 @@ Suggests:
     knitr,
     probably (>= 0.0.6),
     rmarkdown,
-    survival,
+    survival (>= 3.5-0),
```

Comment:
Previous versions of {survival} had a overflow issue when calling `survival::concordance()`